### PR TITLE
Add query param chat init

### PR DESCRIPTION
**Pull request by devceline**
# Changes
- Add QRCode button in `/chats` to create a scannable link so a chat can be started from a phone targetting the desktop. This is not a relay link, it's a normal `https://` link that opens the sample app.

Motivation is to show off possibilities of ease of use and UX

### CODE REVIEW

**Comment by bkrem** [comment]
I see the what from the title but not the "why". Could you add context regarding what this achieves in the PR description please? 🙏

File: wallets/react-wallet-chat/src/pages/newChat.tsx [hunk 1]
```diff
@@ -1,47 +1,72 @@
...
+        account: targetAddress,
+        invite
+      })
+    },
+    [eip155Address]
+  )
+
+  const onInvite = useCallback(
+    async (addressToInvite: string) => {
+      if (demoContactsMap[addressToInvite]) {
```
Comment (bkrem) [inline wallets/react-wallet-chat/src/pages/newChat.tsx:42]: Not in scope for this PR, but we can kill this whole demo mapping at some point. This was just for the MVP/demo video.

**Comment by bkrem** [comment]
The new invite flow via w3mv2 is slick, great stuff ✨ 🚀

**Comment by bkrem** [comment]
Minor UX: would be dope if we automatically close web3Modal when the chat invite comes through (i.e. on `chat_invite`) if it's not overly complicated. 

Initially took me a moment to realise there was a chat invite there until I closed the modal myself.

File: wallets/react-wallet-chat/src/pages/newChat.tsx [hunk 2]
```diff
@@ -1,47 +1,72 @@
...
-  const createInvite = async (targetAddress: string) => {
-    const invite: ChatClientTypes.PartialInvite = {
-      message: "hey let's chat",
-      account: `eip155:1:${eip155Address}`
-    }
-    const inviteId = await chatClient.invite({
-      account: targetAddress,
-      invite
+  useEffect(() => {
+    chatClient.on('chat_joined', args => {
```
[the remark this answers is not in the document]
Comment (devceline) [reply wallets/react-wallet-chat/src/pages/newChat.tsx:20]: Hm, replacing it with `once` would solve this I think

File: wallets/react-wallet-chat/src/pages/newChat.tsx (hunk 1 above)
Comment (devceline) [reply wallets/react-wallet-chat/src/pages/newChat.tsx:42]: For sure, something to tackle soon

**Review by bkrem — APPROVED** [review]
Seems good to go, but not sure if you're done on your side. Feel free to ping me for re-review in situations like this to keep things moving 👍

**Pull request merged**